<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtManagedWidgetsDialog</class>
 <widget class="QDialog" name="QtManagedWidgetsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customize toolbar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtmanagedwidgets.qrc">
    <normaloff>:/qtmanagedwidgets/customize.png</normaloff>:/qtmanagedwidgets/customize.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vlayAvailable">
     <item>
      <widget class="QLabel" name="lblAvailable">
       <property name="text">
        <string>Items available:</string>
       </property>
       <property name="buddy">
        <cstring>listAvailable</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtManagedWidgetsListbox" name="listAvailable">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlayButtons">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="vSpacerTop">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnUp">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToRight">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Add -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToLeft">
       <property name="text">
        <string>&lt;- Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDown">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vSpacerBottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlayVisible">
     <item>
      <widget class="QLabel" name="lblVisible">
       <property name="text">
        <string>Items visible:</string>
       </property>
       <property name="buddy">
        <cstring>listVisible</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listVisible">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtManagedWidgetsListbox</class>
   <extends>QListWidget</extends>
   <header>qtmanagedwidgetslistbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnBox</tabstop>
  <tabstop>listAvailable</tabstop>
  <tabstop>listVisible</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnToRight</tabstop>
  <tabstop>btnToLeft</tabstop>
  <tabstop>btnDown</tabstop>
 </tabstops>
 <resources>
  <include location="qtmanagedwidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>QtManagedWidgetsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>QtManagedWidgetsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
